example-workflows/README.md

```diff
@@ -0,0 +1,115 @@
+# Cloud Functions - GitHub Actions
+
+An example [workflow](.github/workflows/cloud-functions.yml) that uses [GitHub Actions][actions] to deploy a
+[Hello World Python app](main.py) to [Cloud Funtions][cloud-functions].
+
+This code is intended to be an _example_. You will likely need to change or
+update values to match your setup.
+
+## Workflow description
```

Comment:
Please review all steps, these don't match the workflow file.

Reply:
Corrected the steps to align with workflow file.